tests/Makefile

```diff
@@ -53,7 +53,7 @@ NC='\033[0m'
 
 .PHONY: clean list build all
 
-all: framework build 
+all: build
```

Comment:
? you removed framework that's required. it has to happen first.

build should dependent on it.